.github/workflows/pr-deploy.yaml

```diff
@@ -294,16 +328,16 @@ jobs:
           echo "::add-mask::$password"
           echo "password=$password" >> $GITHUB_OUTPUT
 
-          /home/runner/coder login \
-            --first-user-username pr${{ env.PR_NUMBER }} \
-            --first-user-email ${{ env.PR_NUMBER }}@coder.com \
+          coder login \
```

Comment:
nit: use `--global-config` to store the resulting tokens etc. in a well-known location

Reply:
Doesn't it defaults to `.config/coderv2`? Or we should use another?

Reply:
Yes, to `${HOME}/.config/coderv2` - my only thought is to have some method of separating the credentials for each PR env.

Reply:
This is fine as it is run in GitHUb actions runner, and these are isolated and ephemeral for each run.